src/main/java/moze_intel/projecte/gameObjs/tiles/CollectorMK1Tile.java

```diff
@@ -171,76 +173,75 @@ else if (!getUpgrading().isEmpty())
 		}
 	}
 	
-	private void updateEmc()
-	{
-		if (!this.hasMaxedEmc())
-		{
-			this.addEMC(getSunRelativeEmc(emcGen) / 20.0f);
+	private void updateEmc() {
+		if (emcTimer > 0) {
+			emcTimer--;
+		} else {
+			if (!this.hasMaxedEmc()) {
+				this.addEMC(getSunRelativeEmc(emcGen) / 4);
+			}
+			emcTimer = 5;
 		}
```

Comment:
This file has a lot of formatting changes that make reviewing hard

Reply:
I tried to only attach the timers to parts actually generating emc so the rest of the tile can function as normal